docs/02.API-REFERENCE.md

```diff
@@ -2766,6 +2819,59 @@ jerry_resolve_or_reject_promise (jerry_value_t promise,
 - [jerry_release_value](#jerry_release_value)
 - [jerry_value_is_error](#jerry_value_is_error)
 
+# Functions for symbols
+
+These APIs all depend on the ES2015-subset profile.
+
+## jerry_get_symbol_descriptive_string
+
+**Summary**
+
+Call the SymbolDescriptiveString ecma builtin operation on the API value.
+
+*Note*: Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+is no longer needed.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_get_symbol_descriptive_string (const jerry_value_t value);
+```
+
+- `value` - symbol value
+- return value
+  - string value containing the symbol's descriptive string - if success
+  - thrown error, otherwise
+
+**Example**
+
+[doctest]: # ()
+
+```c
+#include "jerryscript.h"
+
+int
+main (void)
+{
+  jerry_init (JERRY_INIT_EMPTY);
+
+  jerry_value_t string_value = jerry_create_string ((const jerry_char_t *) "foo");
+  jerry_value_t symbol_value = jerry_create_symbol (string_value);
+
+  jerry_release_value (string_value);
+
+  jerry_value_t symbol_desc_string = jerry_get_symbol_descriptive_string (symbol_value);
+
+  // usage of symbol_to_string
```

Comment:
AFAIK, this should be: `symbol_desc_string` an not `symbol_to_string`

Reply:
Good catch!